programs/manifest/src/program/processor/swap.rs

```diff
@@ -88,26 +91,43 @@ pub(crate) fn process_swap(
     // net token transfers will be handled later
     dynamic_account.deposit(payer.key, in_atoms, is_base_in)?;
 
+    // 4 cases:
+    // 1. Exact in base. Simplest case, just use the base atoms given.
+    // 2. Exact in quote. Search the asks for the number of base atoms in bids to match.
```

Comment:
Are you making some assumption on what is base vs quote? Could be someone makes a USDC/SOL market or something

Reply:
no